src/analyses/constant_propagator.cpp

```diff
@@ -495,7 +651,31 @@ bool constant_propagator_domaint::merge(
   locationt from,
   locationt to)
 {
-  return values.merge(other.values);
+  const symbol_tablet symbol_table;
+  const namespacet ns(symbol_table);
```

Comment:
Those lines should probably be moved inside the `#if 0` (and then one wonders what the secrets here are).